modules/core/src/test/scala/doobie/util/FragmentsSuite.scala

```diff
@@ -25,87 +25,87 @@ class FragmentsSuite extends munit.FunSuite {
   val ofs  = List(1,2,3).map(n => Some(fr"$n").filter(_ => n % 2 =!= 0))
 
   test("values for one column") {
-    assertEquals(values(nel).query[Unit].sql, "VALUES (?), (?), (?) ")
+    assertEquals(values(nel).query[Unit].sql, "VALUES ?, ?, ? ")
   }
 
   test("values for two columns") {
-    assertEquals(values(NonEmptyList.of((1, true), (2, false))).query[Unit].sql, "VALUES (?,?), (?,?) ")
+    assertEquals(values(NonEmptyList.of((1, true), (2, false))).query[Unit].sql, "VALUES ?,?, ?,? ")
```

Comment:
Similar to above, we definitely want to keep `VALUES (?,?), (?,?)` because we're inserting two tuples no?

Reply:
Fixed :+1: